pandas/core/groupby/groupby.py

```diff
@@ -673,14 +673,14 @@ def f(self):
             return self.plot(*args, **kwargs)
 
         f.__name__ = "plot"
-        return self._groupby.apply(f)
+        return self._groupby._python_apply_general(f, self._groupby._selected_obj)
 
     def __getattr__(self, name: str):
         def attr(*args, **kwargs):
             def f(self):
                 return getattr(self.plot, name)(*args, **kwargs)
 
-            return self._groupby.apply(f)
+            return self._groupby._python_apply_general(f, self._groupby._selected_obj)
```

Comment:
does selected_obj vs obj_with_exclusions matter here?   think we're mostly standardizing on the latter

Reply:
Yes - this is preserving current behavior that will then be deprecated (#7155). Enforcing that deprecation will allow us to use selected_obj.